example.js

```diff
@@ -1,4 +1,3 @@
-const util = require('util')
```

Comment:
Never used

Reply:
Yep, it was there because I've used it pretty print some json values. Fine with this change.